src/components-examples/material/slider/slider-configurable/slider-configurable-example.css

```diff
@@ -4,17 +4,20 @@
 
 .example-section {
   display: flex;
+  flex-wrap: wrap;
   align-content: center;
   align-items: center;
-  height: 60px;
 }
 
 .example-margin {
   margin: 8px;
 }
+.example-width {
+  width: min(calc(100% - 16px), 300px);
```

Comment:
I don't quite follow why we need the `min` and `calc` here. Couldn't we achieve the same with a `max-width`?

Reply:
@crisbeto , I have used `width` and `max-width` this time and added new commit. please review and let me know if any changes required.